test/controllers/DeleteAdditionalMetadataControllerSpec.scala

```diff
@@ -65,6 +71,16 @@ class DeleteAdditionalMetadataControllerSpec extends FrontEndTestHelper {
         .apply(FakeRequest(GET, s"/consignment/$consignmentId/additional-metadata/confirm-delete-metadata/$closureMetadataType"))
       val deleteMetadataPage = contentAsString(response)
 
+      case class GraphqlRequestData(query: String, variables: gcfm.Variables)
```

Comment:
This case class is used with three different sets of tests. Might be worth abstracting it to the `FrontEndTestHelper` class